exercises/complex-numbers/src/complex_numbers.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _COMPLEX_NUMBERS_H_
+#define _COMPLEX_NUMBERS_H_
+
+typedef struct Complex {
+    double real;
+    double imag;
+} Complex;
+
+Complex c_add(Complex* a, Complex* b);
+Complex c_sub(Complex* a, Complex* b);
+Complex c_mul(Complex* a, Complex* b);
+Complex c_div(Complex* a, Complex* b);
+double c_abs(Complex* x);
+Complex c_conjugate(Complex *x);
+double c_real(Complex* x);
+double c_imag(Complex* x);
+Complex c_exp(Complex* x);
```

Comment:
Since none of these should modify the provided number(s), can you mark them as `const`?

Reply:
Why don't we just pass them by value instead? The pointer is already half the size of the struct so it's not like there's a lot of overhead, and it simplifies the interface.

Reply:
I had considered that. I'll pass them by value.